config/locales/defaults/datetime/en.yml

```diff
@@ -26,7 +26,7 @@ en:
     format_string:
       flatpickr: Y-m-d h:i K
       js: YYYY-MM-DD h:mm A
-      placeholder: YYYY-MM-DD H:MM AM/PM
+      placeholder: Leave blank to use assignment due date.
```

Comment:
This is a fairly general string, so you shouldn't change it (instead create a new key).

But also, I think you misunderstood what I asked you to do: I don't want to change the placeholder text of the _assignment_ due date/start time/end time, but rather the placeholder text of the _section-specific inputs_ for these values.